pkg/volumeclaimbinder/types.go

```diff
@@ -75,23 +75,29 @@ type matchPredicate func(compareThis, toThis *api.PersistentVolume) bool
 
 // Find returns the nearest PV from the ordered list or nil if a match is not found
 func (pvIndex *persistentVolumeOrderedIndex) Find(pv *api.PersistentVolume, matchPredicate matchPredicate) (*api.PersistentVolume, error) {
-	volumes, err := pvIndex.ListByAccessModes(pv.Spec.AccessModes)
-	if err != nil {
-		return nil, err
-	}
+	// the user may ask for access modes which either don't exist exactly as specified or can be matched by another mode.
```

Comment:
This comment talks about a user requesting modes, but pv.Spec is the OFFERED modes, isn't it?  This could be clearer.